File: Demo Projects/SQlite_DB/app/src/main/java/com/example/sqlite_db/MainActivity.java
package com.example.sqlite_db;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
            DB_handler handler = new DB_handler(this,"empdb",null,1);
//            handler.addEmployee(new Employee(10,"Ritesh",33.3));
            handler.getEmployee(10);
            handler.close();

    }
}

File: Latitude_projects/Retrofit_Latitude/app/src/main/java/com/example/retrofit_latitude/MainActivity.kt
package com.example.retrofit_latitude

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

import android.provider.Settings.Global
import android.util.Log
import kotlinx.coroutines.GlobalScope
import kotlinx.coroutines.launch
import retrofit2.create

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val quotesApi =RetrofitHelper.getInstance().create(QuotesApi::class.java)
        //launching a new coroutine
        GlobalScope.launch {
            val result=quotesApi.getQuotes()
            if (result != null)
            // Checking the results
                Log.d("ritesh", result.body().toString())
        }
        }
    }
